fineract-provider/src/main/resources/application.properties

```diff
@@ -99,7 +99,7 @@ spring.datasource.hikari.dataSourceProperties['dumpQueriesOnException']=${FINERA
 spring.jpa.open-in-view=false
 
 # Liquibase configuration
-spring.liquibase.enabled=true
+spring.liquibase.enabled=${fineract.mode.write-enabled}
```

Comment:
At some point we have to tell to Spring that Liquibase migration should we skipped, and as far as I am aware this way we can be sure Spring does not run any Liquibase related code. If you have any suggestion where should we put this condition then feel free to share, you are way more familiar with the Liquibase configuration in Fineract than me.

Reply:
Oh okay, I get it now. The default Liquibase migration you're used to is actually disabled because the AutoConfiguration is excluded from the Spring context (see `AbstractApplicationConfiguration`). Essentially the only Liquibase migration done is within the `TenantDatabaseUpgradeService` where you have the check exactly if the instance is a write instance.

The reason the property is there in the application.properties is because the `TenantDatabaseUpgradeService` checks this same property whether the migration should be enabled/disabled.